translations/translations.json

```
@@ -1060,7 +1060,8 @@
       "thanksMessage": "Vielen Dank für Ihr Feedback. Damit helfen Sie uns, unsere Inhalte und die {{appName}}-App zu verbessern. Wir würden uns über weiteres Feedback freuen.",
       "close": "Schließen",
       "negativeRating": "Diese Seite ist nicht hilfreich",
-      "positiveRating": "Diese Seite ist hilfreich"
+      "positiveRating": "Diese Seite ist hilfreich",
+      "contactMailAdress": "Email für eventuelle Rückfragen (optional)"
```

```suggestion
      "contactMailAdress": "E-Mail für eventuelle Rückfragen (optional)"
```
Maybe we should translate `optional` separately and combine it :thinking: